packages/settings-library/src/settings_library/docker_registry.py

```diff
@@ -23,10 +23,26 @@ class RegistrySettings(BaseCustomSettings):
     )
     REGISTRY_SSL: bool = Field(..., description="access to registry through ssl")
 
+    @validator("REGISTRY_PATH", pre=True)
+    @classmethod
+    def escape_none_string(cls, v) -> Optional[str]:
+        return None if v == "None" else v
```

Comment:
Because when you serialise and deserialise to environment variables via string casting `str(None)` the output will be string `"None"`.
This is shared between director-v2 and dynamic-sidecar. director-v2 will serialise to env vars and dynamic-sidecar will read the values back.